### ISSUE TITLE
SE is removing OpenID

### ISSUE THREAD

**Issue by FelixSFD** [Jacob-Gray/ChatExchangeJS#4]
https://meta.stackexchange.com/q/307647/347985

It looks like the chat library is using OpenID for authentication.

https://github.com/Jacob-Gray/ChatExchangeJS/blob/5ce60ef8c301d5ed7f33afb2321c409d805b59b7/ChatExchangeJS/config.js#L11-L13

[This post](https://blog.sobotics.org/2018/07/replacing-openid) explains the new login-flow in Java. It shouldn't be too hard to do the same in JS.

**Comment by Jacob-Gray** [comment]
@FelixSFD Yep, the library is currently broken. I've been too busy to get to it recently, but I might be able to get it in working condition in 2 months or so. That update should also bring some feature updates as well.
